### PR TITLE
feat: add devops category

### PR DESCRIPTION
**Pull request by ghostx31**
## Description

This PR adds the following:
- A DevOps category as was discussed with @Bhupesh-V in the Discord server.
- An entry for Golang under `languages/go`. 

## Changes Include
- [x] New feature ✨ (non-breaking change which adds functionality)

## How Has This Been Tested?
- The `index.json` files were validated by using the `validator.py` script.

### CODE REVIEW

File: devops/index.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      "title": "Linux Journey",
+      "url": "https://linuxjourney.com/",
+      "type": "website",
+      "level": "everyone",
+      "review": [
+        "This is a very good website to gain hands-on experience in Linux and bash."
+      ]
+    },
+    {
+      "title": "Docker",
```
Comment (Bhupesh-V) [inline devops/index.json:28]:
```suggestion
      "title": "Docker Tutorial for Beginners [FULL COURSE in 3 Hours] - TechWorld With Nana",
```

File: devops/index.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      "url": "https://www.youtube.com/watch?v=3c-iBn73dDE",
+      "type": "video",
+      "level": "intermediate"
+    },
+    {
+      "title": "A basic introduction to Kubernetes",
```
Comment (Bhupesh-V) [inline devops/index.json:34]:
```suggestion
      "title": "Kubernetes Tutorial for Beginners [FULL COURSE in 4 Hours] - TechWorld With Nana",
```

**Review by Bhupesh-V — CHANGES_REQUESTED** [review]

**Comment by ghostx31** [comment]
I've made the changes to the titles as requested!

**Comment by Bhupesh-V** [comment]
@all-contributors please add @ghostx31 for content

**Pull request merged**